### DISCUSSION TITLE
import csvs to tables not working

### DISCUSSION THREAD

**Discussion by ilianavalor** [dbeaver/dbeaver#11884] [Q&A]
whenever i try to upload csvs into my tables, i keep getting this error. any ideas on how to fix it?

![image](https://user-images.githubusercontent.com/71728938/112504116-7a655880-8d59-11eb-988b-16b44fe1f8a0.png)

**Comment by LonwoLonwo** [comment]
Hello @ilianavalor 
Need more details.
-DBeaver version
-Database version
-That type of columns is in your table? May be table DDL
- Have you try disable batches?
in import settings 
![2021-03-26 00_53_50-Data Transfer](https://user-images.githubusercontent.com/45152336/112548782-d10c7a00-8dcd-11eb-94fc-530db324f8ad.png)

**Reply by ilianavalor** [reply]
Dbeaver 7.3.3, Redshift database, and the columns of the table match to the existing table.

Yes, I've tried disabling batching and the error still occurs.

Is there a feature within uploading csvs to ignore all parameters?

**Comment by LonwoLonwo** [comment]
Can't reproduce on Redshift in DBeaver 21.0.1
You can try new DBeaver version, where some data transfer problem was fixed https://dbeaver.io/download/
Or you can share here log files https://github.com/dbeaver/dbeaver/wiki/Log-files
